lighthouse-core/config/default.js

```diff
@@ -12,8 +12,17 @@ module.exports = {
       "viewport-dimensions",
       "theme-color",
       "manifest",
+      "chrome-console-messages",
       "image-usage",
-      "accessibility"
+      "accessibility",
+      "dobetterweb/all-event-listeners",
+      "dobetterweb/anchors-with-no-rel-noopener",
+      "dobetterweb/appcache",
+      "dobetterweb/domstats",
+      "dobetterweb/optimized-images",
```

Comment:
in this gatherer, we flip `Network` domain on and off. 

i'm not sure why though.  I suppose its for the `'Network.getResponseBody'` we do. there's a chance this method will work without network being enabled. lets doublecheck that?  looking at response-compression.js, it appears to get response bodies while network is disabled:

![image](https://cloud.githubusercontent.com/assets/39191/25866418/4810c578-34ab-11e7-8ac5-b0b1b64537d4.png)


(also `driver.getRequestContent` exists now which it should probably use?)

it _could_ be a problem because we may allow more network events into the devtoolsLog than what we intended. but yeah... if we dont need to enable/disable in there then let's remove.